packages/cli/src/installers/node/index.ts

```diff
@@ -144,7 +118,80 @@ export async function installNode(pkg: { [key: string]: any }) {
     )
   }
 
-  console.log(displayOutroMessage(pushApiKey, name))
+  const { method } = await inquirer.prompt([
+    {
+      type: "list",
+      name: "method",
+      message:
+        "Which method of configuring AppSignal in your project do you prefer?",
+      choices: [
+        "Using an appsignal.js configuration file.",
+        "Using system environment variables."
+      ],
+      default: "Using an appsignal.js configuration file."
+    }
+  ])
+
+  if (method == "Using an appsignal.js configuration file.") {
+    console.log()
+    console.log(`Writing ${configurationFilename} configuration file.`)
+
+    fs.writeFileSync(
+      path.join(dir, configurationFilename),
```

Comment:
Maybe we should make sure the file doesn't already exist, so we don't accidentally overwrite users' changes?

Reply:
Or prompt the user if they want to overwrite it or not.

Reply:
Done in https://github.com/appsignal/appsignal-javascript/pull/579/commits/1dfec374fae8cac31cdb55dd8a9f3eec9449dfa3.